wetmap/src/components/newModals/tripCreatorListModal/types.ts

```diff
@@ -0,0 +1,13 @@
+export type ItineraryItem = {
```

Comment:
Seems like ItineraryItem type already exists in shopModal. I would suggest to move it to `entities` and avoid duplication.
Plus you have some small formatting issues my VSCode complaining about. Could you please take a look at that